app/routes/settings/exports.js

```diff
@@ -0,0 +1,43 @@
+import Route from 'ember-route';
+import get from 'ember-metal/get';
+import set from 'ember-metal/set';
+import { task } from 'ember-concurrency';
+import Pagination from 'client/mixins/pagination';
+
+export default Route.extend(Pagination, {
+  model() {
+    return {
+      taskInstance: get(this, 'modelTask').perform(),
+      paginatedRecords: []
+    };
+  },
+
+  modelTask: task(function* () {
+    return yield this.queryPaginated('linked-account', {
+      include: 'libraryEntryLogs.media'
```

Comment:
Let's use fieldsets here since we're bringing in media. http://jsonapi.org/format/#fetching-sparse-fieldsets